cli/js/web/util.ts

```diff
@@ -189,3 +189,22 @@ export function defineEnumerableProps(
     Reflect.defineProperty(Ctor.prototype, prop, { enumerable: true });
   }
 }
+
+// @internal
```

Comment:
FYI `@internal` has no effect. There are many all over the code base, so I won't ask you to correct before landing.

Reply:
Thanks! 
I added it since it was all over the place and _went with the flow_. If it's not needed I can submit a PR to remove all `// @internal`, won't be hard to remove them all.